TrackletAlgorithm/TrackHandler.h

```diff
@@ -2,8 +2,8 @@
 #define TrackletAlgorithm_TrackHandler_h
 #include "TrackFitMemory.h"
 #include "hls_stream.h"
-const int NBarrelStub = 4;
-const int NDiskStub = 4;
+const int NBarrelStub = 6;
+const int NDiskStub = 5;
```

Comment:
@aryd Please use the constants from TrackletAlgorithm/Constants.h here:
```suggestion
const int NBarrelStub = trklet::N_LAYER;
const int NDiskStub = trklet::N_DISK;
```
You could also do away with `NBarrelStub` and `NDiskStub` entirely, and replace them with `trklet::N_LAYER` and `trklet::N_DISK`.

Reply:
Fixed.